go.mod

```diff
@@ -46,6 +46,8 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/openshift/api v0.0.0-20240527133614-ba11c1587003 // indirect
+	github.com/openshift/client-go v0.0.0-20240528061634-b054aa794d87 // indirect
```

Comment:
Since there is no new direct pkg addition, why these indirect dependencies are getting added?

Reply:
I have added a direct dependency in common.go

Reply:
Ohh, then could you please move this to the first require block and remove the indirect comment as it is confusing
And then run `go mod tidy`

Reply:
Yes done @ashimagarg27 
Could you PTAL